Dockerfile

```diff
@@ -1,11 +1,16 @@
 FROM docs/docker.github.io:docs-base
 
+# IF THIS IS NOW AN ARCHIVE: remove everything in this file (including the line above) and replace it with the below (and uncomment):
+# (see https://docker.atlassian.net/wiki/spaces/DOCSTEAM/pages/133081331/Docs+release+and+archiving+process):
```

Comment:
This is not accessible to non-members I presume? Is it possible maybe to use a link that's open to the public? :)

Reply:
Hi @axilleas! The archiving process for docs is a Docker-internal process, because we need access to some systems and tools that are not available externally, as well as access to Docker release engineering etc. Sorry about the internal-only link, though.

Reply:
Sure, no problem, if there's no other way!